spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/servicebus/ServiceBusRuntimeException.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.servicebus;
+
+import org.springframework.core.NestedRuntimeException;
+
+/**
+ * The Azure Event Hub specific {@link NestedRuntimeException}.
```

Comment:
The comment here is `Azure Event Hub`, is it same with `Service Bus`?

Reply:
Should be Service Bus